arch/arm64/boot/dts/rockchip/overlays/radxa-cm5-rpi-cm4-io-deskpi-mini-cube.dts

```diff
@@ -0,0 +1,48 @@
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/pwm/pwm.h>
+#include <dt-bindings/thermal/thermal.h>
+
+/ {
+	metadata {
+		title = "Enable DeskPi Mini Cube";
+		category = "misc";
+		compatible = "radxa,cm5-rpi-cm4-io";
+		description = "Enable DeskPi Mini Cube.";
```

Comment:
```suggestion
		description = "Enable support for DeskPi Mini Cube.

Due to hardware limitations, HDMI over the USB Type-C will not work.";
```

We can add multi line description, and this can be viewed in rsetup.

Reply:
Updated and forced-pushed. Used the word "above" instead of "over" because "over" can imply that the signal is electrically going through USB-C, but "above" is more specific.